source/test/E2E.test.js

```diff
@@ -1,98 +1,90 @@
 
-describe('Basic web functionality check', () => {
-
-    jest.setTimeout(5000);
 
+    
 
 
-    it('Check thumnails on home page',async () => {
-        const puppeteer = require("puppeteer");
-        const browser = await puppeteer.launch();
-        const page = await browser.newPage();
-        await page.goto("https://quickbtyes.netlify.app/source/home-page/");
-        
-        let checkMark = true;
-        //wait until the thumnails are generated 
-        await page.waitForSelector("#tile");
-
-        //1. check if we have thumnails
-        const numContainer = await page.$$eval(".container", (prodItems) => {
-            return prodItems.length;
-          });
-        const numThumbnails = await page.$$eval('#tile', (prodItems) => {
 
+test('Check thumnails on home page',async () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Big-Bytez/cse110-fa21-group19/pullRequest?prid=8444953)
```suggestion
test("Check thumnails on home page",async () => {
```